lib/letters.ml

```diff
@@ -176,6 +176,8 @@ let ca_path_peer_verifier path =
   Lwt.return (X509.Authenticator.chain_of_trust ~time:now certs)
 ;;
 
+let cert = ref None
```

Comment:
I wanted to memoize the CA certs fetching, else it is done for each mail send, what do you think? This is under the assumption that the CA certs location does not change while the `letters` process is running.

Reply:
This needs to be documented and made very explicit imo but otherwise fine by me.

Reply:
I added a comment to the `ref` and a phrase in the README.